appveyor.yml

```diff
@@ -0,0 +1,166 @@
+        tar -xzf openexr-2.2.1.tar.gz ; cd openexr* ; md "$OSL_INT_DIR" -Force | Out-Null; cd "$OSL_INT_DIR"
+        cmake ..\.. -G "$GENERATOR" -DCMAKE_CONFIGURATION_TYPES="$env:CONFIGURATION" -DCMAKE_PREFIX_PATH="$DEP_DIR" -DCMAKE_INSTALL_PREFIX="$DEP_DIR" -DILMBASE_PACKAGE_PREFIX="$DEP_DIR"
+        cmake --build . --config $env:CONFIGURATION --target INSTALL
+
+        # OpenImageIO
+        cd $DWN_DIR
+        Start-FileDownload https://github.com/OpenImageIO/oiio/archive/Release-1.8.7.zip
+        unzip Release-1.8.7.zip; cd .\oiio-Release*\; md "$OSL_INT_DIR" -Force | Out-Null; cd "$OSL_INT_DIR"
+        cmake ..\.. -G "$GENERATOR" -DCMAKE_CONFIGURATION_TYPES="$env:CONFIGURATION" -DCMAKE_PREFIX_PATH="$DEP_DIR;$BOOST_ROOT" -DCMAKE_INSTALL_PREFIX="$DEP_DIR" -DFREETYPE_DIR="$DEP_DIR" -DBOOST_LIBRARYDIR="$BOOST_LIBRARYDIR" -DUSE_PYTHON=OFF -DOIIO_BUILD_TESTS=OFF -DUSE_NUKE=OFF
+        cmake --build . --config $env:CONFIGURATION --target INSTALL
+
+        # LLVM
+        cd $DWN_DIR
+        git clone --depth 10 --branch release_40 https://github.com/llvm-mirror/llvm.git
+        cd llvm; md "$OSL_INT_DIR" -Force | Out-Null; cd "$OSL_INT_DIR"
+        cmake ..\.. -G "$GENERATOR" -DCMAKE_CONFIGURATION_TYPES="$env:CONFIGURATION" -DCMAKE_PREFIX_PATH="$DEP_DIR" -DCMAKE_INSTALL_PREFIX="$DEP_DIR" -DLLVM_TARGETS_TO_BUILD="X86"
+        cmake --build . --config $env:CONFIGURATION --target INSTALL
+
+        cd $env:APPVEYOR_BUILD_FOLDER
+    }
+
+    # Always do this
+    cinst winflexbison
+
+
+build_script:
+- ps: |
+    # Build OSL
+    md "$OSL_INT_DIR" -Force | Out-Null; cd "$OSL_INT_DIR"
+    cmake ..\.. -G "$GENERATOR" -DCMAKE_CONFIGURATION_TYPES="$env:CONFIGURATION" -DCMAKE_PREFIX_PATH="$DEP_DIR;$BOOST_ROOT;$LLVM_DIR" -DCMAKE_INSTALL_PREFIX="$DEP_DIR" -DLLVM_DIRECTORY="$LLVM_DIR" -DLLVM_STATIC=ON -DOPENIMAGEIOHOME="$DEP_DIR" -DBOOST_LIBRARYDIR="$BOOST_LIBRARYDIR" -DFLEX_EXECUTABLE="C:\ProgramData\chocolatey\lib\winflexbison\tools\win_flex.exe" -DBISON_EXECUTABLE="C:\ProgramData\chocolatey\lib\winflexbison\tools\win_bison.exe" -DUSE_QT=OFF -DBUILDSTATIC=OFF -DLINKSTATIC=OFF
+
+    # OSL's CMake should probably be handling Windows config better...
+    $OSL_BUILD_DIR = "$env:APPVEYOR_BUILD_FOLDER\$OSL_INT_DIR"
+    $env:Path += ";$BOOST_LIBRARYDIR"
+    $env:OSLHOME = "$env:APPVEYOR_BUILD_FOLDER\src"
+    $env:OPENIMAGEIOHOME = "$DEP_DIR"
+    $env:OIIO_LIBRARY_PATH = "$OSL_BUILD_DIR\src\osl.imageio\$env:CONFIGURATION"
+    $env:Path += ";$OSL_BUILD_DIR\src\oslc\$env:CONFIGURATION"
+    $env:Path += ";$OSL_BUILD_DIR\src\oslinfo\$env:CONFIGURATION"
+    $env:Path += ";$OSL_BUILD_DIR\src\liboslcomp\$env:CONFIGURATION"
+    $env:Path += ";$OSL_BUILD_DIR\src\liboslexec\$env:CONFIGURATION"
+    $env:Path += ";$OSL_BUILD_DIR\src\liboslnoise\$env:CONFIGURATION"
+    $env:Path += ";$OSL_BUILD_DIR\src\liboslquery\$env:CONFIGURATION"
+
+    cmake --build . --config $env:CONFIGURATION
+
+    # Test OSL
+    ctest -C $env:CONFIGURATION -E "texture-udim"
```

Comment:
Why did you exclude texture-udim? What was going wrong or missing for that test?